File: Sheet 3 (Loops 1)/J.py
n = int(input())
mi = 0 ; ch = 0
for i in range(n):
    m , c = input().split()
    m = int(m) ; c = int(c)
    if m > c:
        mi+=1
    if c > m:
        ch +=1
if mi > ch:
    print("Mishka")
elif ch > mi :
    print("Chris")
else:
    print("Friendship is magic!^^")

File: Sheet 3 (Loops 1)/J.cpp
#include <iostream>
using namespace std;

int main()
{
    int t , M = 0 , C = 0;
    cin >> t;
    while(t--){
        int m , c;
        cin >> m >> c;
        if (m > c)M++;
        if (c > m)C++;
    }
    if (M>C)cout << "Mishka";
    else if (C>M)cout << "Chris";
    else cout << "Friendship is magic!^^";
    
        
    return 0;
}
